api/db-cdr/generate-cdr/bq-schemas/achilles_analysis.json

```diff
@@ -0,0 +1,40 @@
+[
```

Comment:
It looks like these schema files are all here:

https://github.com/all-of-us/curation/tree/master/data_steward/resources/fields

Which is the set of schema files used to populate CDR versions.

Are there changes here? 

If there aren't, we can potentially just use those files directly... if there are, we should make sure that the schema here is identical to the schema there apart from whatever we're adding. And we might want to indicate in the file or directory name that they are tweaked versions of standard table schemas.

Reply:
Yes! You want all in github so i went this route.  We can discuss best solution. Some tables get tweaked like concept for count fields.  Some are custom like achilles_results_concept so it gets a bit messy.  We can get a good solution though.  

Question is what needs to be in bq for use by others and what doesnt.  And we should have one system that generates bq dbs available to others that will be released to public.  We can move code around to where it is appropriate.